CARDS-1778: Set up prems-resources

sdumitriu opened the pull request:
The project name, for use with `start_cards.sh`, is `prems` or `cards4prems`.

See the second commit, I'm not sure it's useful, it restricts access to all PII for the patient except the visit location. Didn't test it, it may break the "welcome, here's your next appointment" or "session will expire in X days" code.

sdumitriu committed:
CARDS-1778: Set up prems-resources

sdumitriu committed:
CARDS-1778: Set up prems-resources
Restrict access to PII

marta- reviewed (changes requested):
Clinic sidebar entries are absent.

The clinics should be:
* TGH-ED - Toronto General Hospital Emergency Department
* TGH-AIP - Toronto General Hospital Inpatient
* TW-ED - Toronto Western Emergency Department
* TW-AIP - Toronto Western Inpatient

marta- commented:
> See the second commit, I'm not sure it's useful, it restricts access to all PII for the patient except the visit location. Didn't test it, it may break the "welcome, here's your next appointment" or "session will expire in X days" code.

Once #1172 is merged and configured for PREMs, the screen displaying those two messages will be skipped. I think the commit is good.

marta- committed:
CARDS-1837: Configure the Patient Portal Survey Instructions

marta- committed:
CARDS-1778: Set up prems-resources
Uppercased the app name

marta- committed:
CARDS-1778: Set up prems-resources
Updated the app name in Terms of Use, minor rewording.
Note: There are not the final terms of use. They have been slightly adjusted from the DATA-PRO ToU so the text doesn't look completely off topic.

marta- committed:
CARDS-1778: Set up prems-resources
Updated the wording for the clinic dashboard

marta- committed:
CARDS-1778: Set up prems-resources
Update clinic mapping and add sidebar extensions for clinics

marta- commented:
> Clinic sidebar entries are absent.
> 
> The clinics should be:
> 
> * TGH-ED - Toronto General Hospital Emergency Department
> * TGH-AIP - Toronto General Hospital Inpatient
> * TW-ED - Toronto Western Emergency Department
> * TW-AIP - Toronto Western Inpatient

The clinics were updated and the sidebar extensions were added in 7fe4909.

marta- commented on prems-resources/clinical-data/src/main/resources/SLING-INF/content/Statistics/AIP.xml:
```diff
@@ -0,0 +1,48 @@
+<!--
```
Remove both Statistics nodes

sdumitriu committed:
CARDS-1778: Set up prems-resources
Update to match latest state of CARDS-1775

sdumitriu committed:
CARDS-1778: Set up prems-resources
Removed duplicate quesionnaires now present in modules/patient-portal

sdumitriu committed:
CARDS-1778: Set up prems-resources
Removed statistics

sdumitriu committed:
CARDS-1778: Set up prems-resources
Dashboard improvements:
- fix time display error
- fix score not being displayed
- remove name
- add mrn

sdumitriu committed:
CARDS-1778: Set up prems-resources
Remove Statistics entry from the sidebar

sashaandjic reviewed (approved):
tested

sdumitriu committed:
CARDS-1778 / CARDS-1782: Configure notification/reminder emails

sdumitriu committed:
CARDS-1778: Set up prems-resources
Fixed wrong time displayed on the clinic dashboard

sdumitriu committed:
CARDS-1778: Set up prems-resources
Make the new project supported by docker

marta- committed:
CARDS-1778: Set up prems-resources
* Deny access to visit location for patient users to prevent an odd-looking alert from being displayed
* Bug fix: fallback to DEFAULT_INSTRUCTIONS for unspecified survey instructions doesn't really work
* UX improvement: do not display the welcome message on the screen informing the patient that there are no pending surveys
* UX improvement: provide a close button on the screen informing the patient there are no pending surveys

marta- committed:
CARDS-1778: Set up prems-resources
Updated welcome message
